manifests/reposync-crd.yaml

```diff
@@ -1296,6 +1296,38 @@ spec:
                     format: int64
                     minimum: 0
                     type: integer
+                  logLevels:
+                    description: logLevels allows one to override the log verbosity
+                      of a reconciler deployment container. Each entry must contain
+                      the name of a reconciler deployment container and the desired
+                      log level.
+                    items:
+                      description: ContainerLogLevelOverride allows to override the
```

Comment:
Don't use the internal struct name in the description here

Reply:
it looks like this is taken from the comment in https://github.com/GoogleContainerTools/kpt-config-sync/pull/677/files#diff-f2a8c540eb1da55be20cd847621833115dd35f755304786bda8a4ff063ce9ad6R100 which follows the same pattern in https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/manifests/reposync-crd.yaml#L309. How should I update the description here?

Reply:
I think the comment has to start with the struct name, otherwise it will fail the presubmit